backend/packages/Upgrade/src/api/errors/UnprocessableEntity.ts

```diff
@@ -0,0 +1,7 @@
+import { HttpError } from 'routing-controllers';
+
+export class UnprocessableEntityError extends HttpError {
```

Comment:
what would be preferable? I throw this when the experiment can't be saved because of maybe unknown Mooclet server issues so I don't have to throw a 500 server error.

Reply:
I mean, this is not getting used anywhere. Have you created this for future use?

Reply:
ah, i thought i was using it, but I had imported and used import { UnprocessableEntityException } from '@nestjs/common' instead! it works the same apparently, I guess that's just as good so I'll delete this